### ISSUE TITLE
Double counting in users?

### ISSUE THREAD

**Issue by albertocottica** [edgeryders/discourse#136]
I tried making a platform-wide social network of ER. To my surprise, I have _two_ connected components:

![image](https://user-images.githubusercontent.com/1976806/37801350-94aa4682-2e26-11e8-8b7f-68cbda9be6ec.png)

Winnie, Nadia and I appear in both components: the skinny one on the right is clearly biofabforum. This should not happen, unless we are identified by different ids in the two different forums. @damingo, is that the case?

**Comment by damingo** [comment]
No, that's not the case. It's exactly the same database - only the UI is different.

**Comment by albertocottica** [comment]
Actually, the problem is somewhere else. I just realized that, when I query the APIs for top-level categories of edgeryders.eu, the biofabfroum.org cats do not even appear. I get 7 in all:

```
campfire
opencare
openvillage
future-makers
docs
knowledge
workspaces
```
So it's kind of a mystery why the damn thing has two Winnies, two Albertos etc. 

However, there is a potentially more serious issue: the `biofab` top level cat does not show up when calling `/categories.json`. It _does_ show up when calling `/site.json`. I am not sure what the implications of this are. For my code, things are easy to solve.

Whatever happens, we need full API access to whitelabelled sites. Winnie, rightly, wanted OpenEthno to work on Biofab!

**Comment by damingo** [comment]
The `biofab` category is now included in `/categories.json`. It was removed from the JSON view because the option `Suppress this category from the homepage.` was set in https://edgeryders.eu/c/biofab -> Edit -> Settings. This is no longer needed as the category is now hidden from the homepage by means of CSS.
